pkg/suggestion/v1alpha3/chocolate/base_chocolate_service.py

```diff
@@ -48,23 +56,32 @@ def create_optimizer(self, algorithm_name):
                 chocolate_search_space[key] = choco.choice(
                     [idx for idx, _ in enumerate(param.list)])
 
+        if algorithm_name in DEPRECATED_ALGORITHM_NAME:
+            warnings.warn(
+                "Algorithm name '{}' is deprecated. Please use '{}'.".format(
+                    algorithm_name, DEPRECATED_ALGORITHM_NAME[algorithm_name],
+                ),
+                DeprecationWarning,
+            )
+            algorithm_name = DEPRECATED_ALGORITHM_NAME[algorithm_name]
+
         # Refer to https://chocolate.readthedocs.io/tutorials/algo.html
         if algorithm_name == "grid":
             self.chocolate_optimizer = choco.Grid(
                 self.conn, chocolate_search_space, clear_db=True)
         # hyperopt-random is the default option in katib.
-        elif algorithm_name == "chocolate-random":
+        elif algorithm_name == "random":
             self.chocolate_optimizer = choco.Random(
                 self.conn, chocolate_search_space, clear_db=True)
-        elif algorithm_name == "chocolate-quasirandom":
+        elif algorithm_name == "quasirandom":
             self.chocolate_optimizer = choco.QuasiRandom(
                 self.conn, chocolate_search_space, clear_db=True)
-        elif algorithm_name == "chocolate-bayesian-optimization":
+        elif algorithm_name == "bayesian-optimization":
```

Comment:
Let's name it `bayesianoptimization` to be consistent with skopt: https://github.com/kubeflow/katib/blob/master/pkg/suggestion/v1alpha3/skopt_service.py#L27